src/pydap/parsers/dmr.py

```diff
@@ -40,7 +41,7 @@ def dap4_to_numpy_typemap(type_string):
     return np.dtype(dtype_str)
 
 
-def get_variables(node, prefix=""):
+def get_variables(node, prefix="") -> dict:

```

Comment:
Do you plan to add type qualifiers to all the methods? Just curious since I've used them and found them useful, but I can see that adding them could be a _huge_ effort. If you do plan on that, my suggestion would be to make it a separate PR.

Reply:
Yes, I think it would be very useful to add the type qualifiers to the methods on a separate PR as you mention.